docs/articles_en/documentation/openvino-ir-format/operation-sets/operations-specifications/movement/strided-slice-1.rst

```diff
@@ -14,6 +14,69 @@ StridedSlice
+
+.. note:: Negative Values in Begin and End (Negative Values Adjusting)
+
+   Negative values represent indexing from the back, i.e., the value of -1 represents the last element of the input dimension. In practice, negative values are automatically incremented by the size of the dimension. For example, if :math:`data = [0, 1, 2, 3]`, :math:`size(data) = 4`, :math:`begin(i) = -1` for some i, this value will be modified to be :math:`begin(i) = -1 + 4 = 3`. Note that if :math:`begin(i) = -5` for some i, this value will be adjusted as follows: :math:`begin(i) -5 + 4 = -1`, which will trigger value clamping.
+
+.. note:: Indexing in Reverse
+
+   If :math:`stride[i] < 0`, the indexing will happen in reverse. At each step, the value of :math:`stride[i]` will be subtracted from the :math:`slicing\_index`. As long as the :math:`slicing\_index > end[i]`, the corresponding element is added to the output. Whenever :math:`slicing\_index <= end[i]`, the slicing stops.
+
+.. note:: Value Out-of-Bounds (Silent Clamping)
+
+   If a value in begin or end is out of bounds for the corresponding dimension, it is silently clamped. In other words:
+
+   * If :math:`begin[i] >= size(dim)`, then :math:`begin[i] = size(dim)`. If :math:`begin[i] < 0` (after Negative Values Adjusting), then :math:`begin[i] = 0`.
+   * If :math:`end[i] >= size(dim)`, then :math:`end[i] = size(dim)`. If :math:`end[i] < 0` (after Negative Values Adjusting), then :math:`end[i] = 0`.
+
+   If slicing in reverse, the clamping behavior changes to the following:
+
+   * If :math:`begin[i] >= size(dim)`, then :math:`begin[i] = size(dim) - 1`. If :math:`begin[i] < 0` (after Negative Values Adjusting), then :math:`begin[i] = 0`.
+   * If :math:`end[i] >= size(dim)`, then :math:`end[i] = size(dim)`. If :math:`end[i] < 0` (after Negative Values Adjusting), then :math:`end[i] = -1`.
+
+The operation accepts multiple bitmasks in the form of integer arrays to modify the above behavior. **If the length of the bitmask is less than the length of the corresponding input, it is assumed that the bitmask is extended with zeros.**
+
+During the i-th slicing step:
+
+* If the :math:`begin\_mask[i]` is set to one, the value of :math:`begin[i]` is set to 0 (size(dim) - 1 if slicing in reverse).
+* If the :math:`end\_mask[i]` is set to one, the value of :math:`end[i]` is set to size(dim) (0 if slicing in reverse - note that this does not allow slicing inclusively with the first value).
+* If the :math:`new\_axis\_mask[i]` is set to one, the values of :math:`begin[i]`, :math:`end[i]`, and stride[i] ARE IGNORED, and a new dimension with size 1 appears in the output. No slicing occurs at this step.
+* If the :math:`shrink\_axis\_mask[i]` is set to one, the values of :math:`begin[i]` **MUST EQUAL** :math:`end[i]` (Note that this would normally result in a size 1 dimension), and the **stride[i]** value IS IGNORED. The corresponding dimension is removed, with only a single element from that dimension remaining.
```

Comment:
> with only a single element from that dimension remaining

Again, It works only for the las dimension

Reply:
See comment above